### ISSUE TITLE
Non-EWW browsers

### ISSUE THREAD

**Issue by kennethbruskiewicz** [charlesroelli/org-board#28]
I noticed that org-board tries to use the system browser if 'eww is not used as the default.

Could this option be expanded so other emacs-based browsers could be used, like w3m?

**Comment by charlesroelli** [comment]
Yes, it should be possible to do this by adapting the expression that calculates the value of `org-board-default-browser`.

**Comment by kennethbruskiewicz** [comment]
Is there a reason not to pass in a browse function directly?

**Comment by charlesroelli** [comment]
Sure @kennethbruskiewicz, you should be able to use any browse function as the value of `org-board-default-browser`. Unfortunately, though, its `:type` is a bit limited at the moment, so you may need to set the variable directly with something like: `(setq org-board-default-browser (quote w3m))` in your Emacs init file.
